editor/sidebar/post-visibility/index.js

```diff
@@ -71,6 +74,45 @@ class PostVisibility extends Component {
 		this.setState( { opened: false } );
 	}
 
+	handleBlur( event ) {
+		if ( this.state.opened && event.relatedTarget !== null ) {
+			const wrapper = findDOMNode( this );
```

Comment:
This is a case where we could avoid `findDOMNode` by instead assigning a `ref` to the rendered element.

https://facebook.github.io/react/docs/refs-and-the-dom.html

>If you have absolutely no control over the child component implementation, your last option is to use `findDOMNode()`, but it is discouraged.